cpp/models/abm/location.h

```diff
@@ -178,13 +194,55 @@ class Location
         return m_cells;
     }
 
+    int get_population()
+    {
+        return m_num_persons;
+    }
+
+    CustomIndexArray<double, AgeGroup, VaccinationState> get_cached_exposure_rate()
+    {
+        return m_cached_exposure_rate;
+    }
+
+    /**
+ * Set the capacity of the location in person and volume
+ * @param persons maximum number of people that can visit the location at the same time
+ * @param volume volume of the location in m^3
+ */
+    void set_capacity(int persons, int volume)
+    {
+        m_capacity.persons = persons;
+        m_capacity.volume  = volume;
+    }
+
+    /**
+ * @return the capacity of the location in person and volume
+ */
+    LocationCapacity get_capacity()
+    {
+        return m_capacity;
+    }
+
+    /**
+ * @param capacity_adapted_transmission_risk if true calculates the relative risk based on the capacity of the location
+ * @return the relative risk factor for the location
+ */
+    double compute_relative_transmission_risk();
+
+    void capacity_adapted_transmission_risk_flag()
+    {
+        m_capacity_adapted_transmission_risk = true;
+    }
```

Comment:
either name th function consider_adapted.... or put an input boolean to set m_capacity_adapted_transmission_risk to true.
Else: Shouldn't m_capacity_adapted_transmission_risk be set to false as default? It isn't.